index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <header>
+      <h1 id="title">freeCodeCamp Survey Form</h1>
+      <p id="description">
+        Thank you for taking the time to help us improve the platform
+      </p>
+    </header>
+    <div class="container">
+      <!-- adding a form Survey -->
+      <form action="" id="survey-form">
+        <div>
+          <label id="name-label" for="name">Your name : </label>
+          <input class="inputs"
+            type="text"
+            name=""
+            id="name"
+            placeholder="Enter your name"
+            required
+          />
+        </div>
+        <div>
+          <label id="email-label" for="email">Your email : </label>
+          <input class="inputs"
+            type="email"
+            name=""
+            id="email"
+            placeholder="Enter your email"
+            required
+          />
+        </div>
+        <div>
+          <label id="number-label" for="number">Enter number : </label>
+          <input class="inputs"
+            type="number"
+            name=""
+            id="number"
+            min="9"
+            max="12"
+            placeholder="Enter your number"
+            required
+          />
+        </div>
+        <div>
+          <label for="dropdown">Which favorite programming language?</label>
+          <div class="select">
+            <select name="" class="inputs" id="dropdown">
+              <option disabled>Choose a programming language</option>
+              <option value="JavaScript">JavaScript</option>
+              <option value="Python">Python</option>
+              <option value="Ruby">Ruby</option>
+              <option value="PHP">PHP</option>
+            </select>
+          </div>
+        </div>
+        <div>
+          <label for="">What is your favorite Framework?</label>
+          <div>
+            <input class="choose" type="radio" name="Angular" id="" />
+            <label for="Angular">Angular</label>
+          </div>
+          <div>
+            <input class="choose" type="radio" name="Laravel" id="" />
+            <label for="Laravel">Laravel</label>
+          </div>
+          <div>
+            <input class="choose" type="radio" name="Django" id="" />
+            <label for="Django">Django</label>
+          </div>
+          <div>
+            <input class="choose" type="radio" name="Ruby on Rails" id="" />
+            <label for="Ruby on rails">Ruby on Rails</label>
+          </div>
+        </div>
+        <div>
+          <label for="">What are trending technologies in your opinion?</label>
+
```

Comment:
For the purpose of readability, you should not leave an empty line like this one. Make sure that your code has a good indentation. And one way to help you have a very nice looking code if you are using VS Code is to use an extension called Prettier.